### PR TITLE
Implement the new design of the `EmailConfirmation` page (#1889)

### LINKED ISSUE #1889: Implement the new design of  `EmailConfirmation` page in the TodoTemplate project
we need to improve the `EmailConfirmation` page based on the new UI/UX design.

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #1889

### CODE REVIEW

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor [hunk 1]
```diff
@@ -14,37 +14,40 @@
...
-        <h1 class="form-title">Email Confirmation</h1>
-        <BitMessageBar Class="form-message-bar"
-                    MessageBarType="@EmailConfirmationMessageType"
-                    OnDismiss="() => EmailConfirmationMessage = null">
-            @EmailConfirmationMessage
-        </BitMessageBar>
         @if (EmailConfirmed)
         {
+            <h1 class="form-title">
+                Email Confirmation
```
Comment (maryamhdr) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor:20]: Plz use this text instead:
`Email Confirmed Successfully!`

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs [hunk 2]
```diff
@@ -62,7 +55,7 @@ await HttpClient.PostAsJsonAsync("Auth/SendConfirmationEmail", new()
 
             EmailConfirmationMessageType = BitMessageBarType.Success;
 
-            EmailConfirmationMessage = "The confirmation link has been re-sent.";
+            EmailConfirmationMessage = "The confirmation email has been re-sent.";
```
Comment (maryamhdr) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs:58]: plz change the text to `The confirmation link has been re-sent to your email address.`

**Review by maryamhdr — DISMISSED** [review]

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs [hunk 3]
```diff
@@ -24,15 +24,8 @@ protected override async Task OnInitAsync()
     {
         await base.OnInitAsync();
 
-        if (EmailConfirmed)
+        if (EmailConfirmed is false)
```
Comment (msynk) [inline src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs:27]: don't we need to handle the `else` part?

**Comment by msynk** [comment]
@sajadkardel 
any updates?

**Comment by msynk** [comment]
@Cyrus-Sushiant 
plz review

@maryamhdr 
plz re-review

File: src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs (hunk 3 above)
Comment (sajadkardel) [reply src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs:27]: We have two pages in design :

1.  Email Confirmation page
2. Email Confirmation Failed page

The page known as the `EmailConfirmation`  is actually part of the sign-up page which notifies the user that an email has been sent.

The `EmailConfirmationFailed`  is for when the user clicks on the link sent in the email and returns to the program. 
There is a condition on this page that checks the validity of the email link.
If it was wrong, we will show the user the design implemented for the email confirmation failed page.
But when the link is correct, only a simple button is displayed to direct the user to sign-in (We have nothing for this in design pages)

However, I think saying an explanatory text is enough to apply to this page as failed page reciprocal term...
@msynk What do you think?

Comment (msynk) [reply src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs:27]: are you sure about this information?
AFAIK there is no `EmailConfirmationFailed` page in the project. and the `EmailConfirmation` page is not part of the signup page and will show after clicking on the link provided in the email (it actually gets redirected by the server to this page).

**Review by Cyrus-Sushiant — APPROVED** [review]

**Comment by msynk** [comment]
@maryamhdr 
plz re-review

**Review by msynk — APPROVED** [review]

**Pull request merged**